README.md

```diff
@@ -130,6 +130,7 @@ Internal permissions that applications generally should not use directly:
 | FingerprintSensor |
 | Microphone |
 | Notifications |
+| PinQuery |
```

Comment:
Do you mean the term pin query? This is an internal permission that no application should use thus we can rename it any day if there is some collision. Then again, I'm not sure there will be anything else that should use word pin. I think we have tried to keep it so that pin is always referring to SIM pin and anything else is code or something like that.

Reply:
Yes. PIN is "personal identification number" which is rather generic term and used also outside sim pin context. So if it were shown to user it might be confusing. But as it is used on internally without it being visible to users -> no need to change / anything.